### ISSUE TITLE
Accessibility: Redundant Label Instructions in ms-button

### ISSUE THREAD

**Issue by Khushveerkaur** [OfficeDev/office-ui-fabric-react#4026]
### Bug Report

- __Package version(s)__: Latest
- __Browser and OS versions__: NA

#### Describe the issue:
One of the ms-button in our page has redundant label instructions leading to accessibility testing failures. The button has aria-label set as the title, thereby leading to redundancy.

![temp1518089251834](https://user-images.githubusercontent.com/36656468/36414421-38097d08-1648-11e8-9642-498521dcc898.png)

**Comment by natalieethell** [comment]
Correct me if I'm wrong, but it looks like if we have visible text labeling the element, the best alternative to aria-label is aria-labelledby (as stated in the spec here: https://www.w3.org/TR/wai-aria/#aria-label). @Khushveerkaur it looks like we don't have a default aria label, so when you use a Fabric Button, instead of passing in your own aria label, you could pass in aria-labelledby. Would this work for you?

**Comment by KevinTCoughlin** [comment]
I think the issue here was that `aria-label` was equal to `title` and therefore redundant. You can pass in custom `title` or `aria-label` text to avoid that redundancy. I'm going to close this as its not reproducible on the Fabric site and there are a number of ways to fix it during implementation in an app.
